parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -146,6 +146,7 @@ public static enum JobSummaryLevel {
   public static final String MAX_PADDING_BYTES    = "parquet.writer.max-padding";
   public static final String MIN_ROW_COUNT_FOR_PAGE_SIZE_CHECK = "parquet.page.size.row.check.min";
   public static final String MAX_ROW_COUNT_FOR_PAGE_SIZE_CHECK = "parquet.page.size.row.check.max";
+  public static final String VALUE_COUNT_FOR_PAGE_SIZE_CHECK = "parquet.page.size.count.check";
```

Comment:
```suggestion
  public static final String VALUE_COUNT_FOR_PAGE_SIZE_CHECK = "parquet.page.size.value.count.check";
```

Reply:
Please also update the doc here: https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/README.md